packages/zosfiles/__tests__/cli/create/ussDir/__snapshots__/ussDir.definition.unit.test.ts.snap

```diff
@@ -29,33 +29,3 @@ Array [
   },
 ]
 `;
-
```

Comment:
Were these obsolete?

Reply:
Yeah i did -u  on the unit test script so it must have auto removed this